DigitalLearningSolutions.Data.Migrations/202303071200_AddUniqueConstraintToCandidateAssessmentSupervisors.cs

```diff
@@ -0,0 +1,47 @@
+﻿using FluentMigrator;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace DigitalLearningSolutions.Data.Migrations
+{
+    using FluentMigrator;
+
+    [Migration(202303071200)]
+    public class AddUniqueConstraintToCandidateAssessmentSupervisors : Migration
+    {
+        public override void Down()
+        {
+            Delete.UniqueConstraint(
+                "IX_CandidateAssessmentSupervisors_CandidateAssessmentID_SupervisorDelegateId_SelfAssessmentSupervisorRoleID"
+            )
+            .FromTable("CandidateAssessmentSupervisors");
+        }
+
+        public override void Up()
+        {
+            Execute.Sql(@"DELETE CAS
+                          FROM CandidateAssessmentSupervisors CAS
```

Comment:
We may have an issue here if there are related rows in CandidateAssessmentSupervisorVerifications which has a CandidateAssessmentSupervisorID FK to CandidateAssessmentSupervisors.ID. Can we update these first to set the ID to the CandidateAssessmentSupervisors record that we will be keeping?

Reply:
Implemented the same.